src/containers/Tenant/i18n/en.json

```diff
@@ -36,5 +36,11 @@
   "actions.alterTable": "Alter table...",
   "actions.alterTopic": "Alter topic...",
   "actions.selectQuery": "Select query...",
-  "actions.upsertQuery": "Upsert query..."
+  "actions.upsertQuery": "Upsert query...",
+  "actions.createDirectory": "Create directory",
+  "schema.tree.dialog.placeholder": "Relative path",
+  "schema.tree.dialog.header": "Create directory",
+  "schema.tree.dialog.description": "Inside directory",
```

Comment:
As for me, just "Inside" will be enough. Otherwise we get "Create directory inside directory".